Description: Document that ESC key closing the help window
 Users were expecting that pressing ESC should close the help window

---
Origin: upstream, https://github.com/Tomas-M/iotop/commit/383443d650bb29bfb7187cae98e21faa40cbf977
Forwarded: not-needed
Last-Update: 2021-09-06

--- iotop-c-1.17.orig/iotop.8
+++ iotop-c-1.17/iotop.8
@@ -196,6 +196,9 @@ Sort by next column
 \fB<left>\fR
 Sort by previous column
 .TP
+\fB<esc>\fR
+Cancel ionice or filter selection. In case only the help window is open then close it
+.TP
 \fBo\fR, \fBO\fR
 Toggle showing only processes with IO activity
 .TP
